Gemfile.lock

```diff
@@ -88,16 +88,16 @@ GEM
       activerecord (>= 4.0)
     awesome_print (1.9.2)
     aws-eventstream (1.3.0)
-    aws-partitions (1.900.0)
```

Comment:
It looks as though the merge with master has downgraded some packages that had already been updated. Try rebasing again and check the versions of these packages in master first. It might be that this commit can be removed completely or, at least, reduced to something simpler.

Reply:
Thanks @jrmhaig. I didn't spot that! I shall merge each dependabot PR separately, as Mark mentioned. Perhaps that will also resolve it. :)